Makefile

```
@@ -0,0 +1,32 @@
+KUEUE_REPO=https://github.com/kubernetes-sigs/kueue.git
+KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
+
+
+.PHONY: install install_kjob install_kueuectl install_gcloud check_python
+
+install: install_kjob
+	pip install .
+
+install_kjob: install_kueuectl
```

```suggestion
install_kjob: install_kubectl
```
It's not required to have kueue to install kjob. But we need to have kubectl.